src/Data/Bool/Properties.agda

```diff
@@ -745,6 +745,50 @@ if-float : ∀ (f : A → B) b {x y} →
 if-float _ true  = refl
 if-float _ false = refl
 
+if-idemp : ∀ b {x : A} →
```

Comment:
Normally we abbreviate idempotent to `idem`

Reply:
Ah and this is again rather an `if-idem-then` and there is also an `if-idem-else`.